lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -1628,7 +1628,14 @@ void request_initialize(const llvm::json::Object &request) {
   body.try_emplace("supportsEvaluateForHovers", true);
   // Available filters or options for the setExceptionBreakpoints request.
   llvm::json::Array filters;
+  std::string triple =
+      std::string(g_dap.debugger.GetSelectedPlatform().GetTriple());
   for (const auto &exc_bp : g_dap.exception_breakpoints) {
+    // Skipping objc breakpoint filters if not working on macos.
```

Comment:
GNUStep is still a thing.  It's likely the exception throw breakpoint is the only part of our ObjC support that would work with GNUStep, however, so maybe this doesn't matter all that much.